trickle/trickle_publisher.go

```diff
@@ -126,11 +122,7 @@ func (c *TricklePublisher) Close() error {
 	if err != nil {
 		return err
 	}
-	resp, err := (&http.Client{Transport: &http.Transport{
-		DisableKeepAlives: true,
-		// ignore orch certs for now
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-	}}).Do(req)
+	resp, err := httpClient().Do(req)
```

Comment:
just checking it was intentional to create a new client on this one

Reply:
yep - this method is not on the hot path and actually only gets invoked if there is a _problem_ sending the segment. So it seems best to have a fresh client here to mitigate any issues with the existing connection, similar to why we reset the client when we retry ([here](https://github.com/livepeer/go-livepeer/pull/3470/files/d11a1572d1a408281f9518a799f595c6c9c79059#diff-511eab9391997233cc775bb636a3093f5bd9a49301d9e395342a021d301a81e8R174))

Reply:
Added some comments clarifying this, thanks - c3330b0e61dc7d94fa0e9cbea9f744e0ac5b5c5f